.github/workflows/build.yml

```diff
@@ -51,3 +51,73 @@ jobs:
           use-cross: true
           command: build
           args: --release --target ${{ matrix.target }} --manifest-path bindings_ffi/Cargo.toml --target-dir bindings_ffi/target
+
+      - name: Upload binary
```

Comment:
We need these artifacts only briefly until the `package` job bellow is able to collect them all and put them into the zip archive.